lib/ui/semantics/local_context_action.cc

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/semantics/local_context_action.h"
+
+namespace blink {
```

Comment:
Why is this inside namespace blink? Is this somehow related with Chrome's blink?

Reply:
matches the rest of the semantics objects which are also in `blink`, I believe we inherited this from chrome